pgd_splicer/tests.py

```diff
@@ -329,3 +337,61 @@ def test_crosscheck_cullpdb(self):
 
         good_out = MonkeyPatch.sitefile('cullpdb_crosscheck.txt')
         self.assertEqual(test_out, file(good_out).read())
+
+
+from unittest import TestCase as TCase
+from Bio.PDB import *
+
+PRO_MIN = -1
+PRO_MAX = 3
+FIELDS = ['a1','a2','a3','a4','a5','a6','a7','L1','L2','L3','L4','L5','phi','psi','ome','chi1','chi2','chi3','chi4','chi5','bm','bs','bg','h_bond_energy','zeta']
+FIELDS_DICT = {}
+
+for i in range(1, len(FIELDS)+1):
+    #shift values into decimels
+    FIELDS_DICT[FIELDS[i-1]] = i*.01
+
+
+class OccTest(TCase):
+
+    def yield_altid(self, alt_structure) :
+        for ichain in alt_structure[0] :
+            for ires in ichain :
+                if ires.is_disordered():
+                    res_sum = {}
+                    count_sum = {}
+                    for atom in ires :
+                        if atom.is_disordered():
+                            try :
+                                res_sum[ atom.get_altloc() ] = res_sum[atom.get_altloc()] + atom.get_occupancy()
+                            except KeyError :
+                                res_sum[atom.get_altloc()] = atom.get_occupancy()
+                            try :
+                                count_sum[atom.get_altloc()] = count_sum[atom.get_altloc()]+1
+                            except KeyError :
+                                count_sum[atom.get_altloc()] = 1
+
+                    avg_dict = {}
+                    for k,v in res_sum.iteritems() :
+                        avg_dict[k] = res_sum[k]/count_sum[k]
+
+                    keepAltID = max(k for k, v in avg_dict.iteritems())
+                    yield keepAltID
+
+
+    def test_compare_struccture(self):
+        structure = PDBParser().get_structure('pdbname',
+                                                      '/opt/pgd/pgd_splicer/testfiles/not-ordered.pdb')
```

Comment:
Absolute pathnames are bad here.  Look at how the other files are referenced for examples.